tests/deployment_utils.py

```diff
@@ -220,18 +220,14 @@ def _create_temp_config(self, config: str):
             f.write(config)
         return file_path
 
-    def run(self, config: str, force_installed: bool = False, vcs_type: str = "none",
-            additional_parameters="", environment=None, expected_to_fail=False, workdir=None):
+    def run(self, config: str, vcs_type: str = "none", workdir=None,
+            additional_parameters="", environment=None, expected_to_fail=False):
 
-        if utils.is_pycharm():
-            cmd = "{0}/universum/__main__.py".format(self.working_dir)
-        else:
-            cmd = "coverage run --branch --append --source='{0}' '{0}/universum/__main__.py'" \
-                .format(self.working_dir)
-
-        # We cannot collect coverage from installed module, so we run it only if specifically told so
-        if force_installed:
+        # We cannot collect coverage from installed module
+        if utils.is_pycharm() or workdir:
```

Comment:
With this change the `workdir` is only used for launching the installed version.
While this unification is generally ok, it should be explicitly expressed in the code.